cli/src/main/resources/logback.xml

```diff
@@ -34,6 +34,7 @@
     <logger name="org.ossreviewtoolkit.analyzer.managers.Yarn2" level="INFO" />
     <logger name="org.ossreviewtoolkit.clients.fossid.FossIdRestService" level="INFO" />
     <logger name="org.ossreviewtoolkit.reporter.reporters.fossid.FossIdReporter" level="INFO" />
+    <logger name="org.ossreviewtoolkit.plugins.reporters.fossid.FossIdSnippetReporter" level="INFO" />
```

Comment:
While touching this, mind also fixing the line above in all `logback.xml` files in a preceding commit?

Reply:
> in all `logback.xml` files in a preceding commit?

Emphasis was on *all*, you missed `helper-cli/src/main/resources/logback.xml`.